radicle/src/storage/git/transport.rs

```diff
@@ -102,7 +99,8 @@ impl git2::transport::SmartSubtransport for Smart {
             Ok(stream)
         } else {
             Err(git2::Error::from_str(&format!(
-                "repository {id} does not have an associated stream"
+                "repository {} does not have an associated stream",
```

Comment:
`<rant>` I can't believe they didn't allow for `{url.id}` as part of that improvement `</rant>`

Reply:
I know, when I realized that I was like "oh, so actually this won't change my life".